go/private/sdk.bzl

```diff
@@ -231,7 +231,7 @@ def _local_sdk(ctx, path):
     for entry in ["src", "pkg", "bin", "lib"]:
         ctx.symlink(path + "/" + entry, entry)
 
-def _sdk_build_file(ctx, platform, version):
+def _sdk_build_file(ctx, platform, version, extra_sdk_version = None):
```

Comment:
We could set this to `remote` by default, which would make it possible to extract the `config_setting` into `BUILD.sdk.bazel`.